terraform/modules/aws-network-load-balancer/resources.lb.tf

```diff
@@ -0,0 +1,106 @@
+resource "aws_lb" "nlb" {
+  name = "${var.environment}-loadbalancer"
+
+  internal                         = false
+  load_balancer_type               = "network"
+  enable_cross_zone_load_balancing = true
+
+  subnets = var.subnet_ids
+
+  tags = {
+    Environment = var.environment
+  }
+}
+
+
+
+resource "aws_lb_listener" "ingress-http" {
+  load_balancer_arn = aws_lb.nlb.arn
+
+  port     = 80
+  protocol = "TCP"
+
+  default_action {
+    type             = "forward"
+    target_group_arn = aws_lb_target_group.nodes-http.arn
+  }
+}
+
+
+resource "aws_lb_target_group" "nodes-http" {
+  name = "${var.environment}-nodes-http"
+
+  vpc_id = data.aws_vpc.this.id
+
+  # NOTE: "instance" - as an alternative type - didn't work at the time this was developed
```

Comment:
Is there an issue about this upstream, or how do we determine when this'll get fixed?

Reply:
No because I didnt care to investigate yesterday. Anyway, I just looked into it. It appears that it's not a bug. Please, ee the adjusted comment.

Reply:
Thanks for doing the research, and adding it to the comments. It'll make this easier for others to understand.